database/migrations/3_createEnvironmentsTable.go

```diff
@@ -16,6 +16,9 @@ func createEnvironmentsTable() *migrate.Migration {
         name varchar(255) NOT NULL,
         slug varchar(255) NOT NULL,
         application_id bigint(20) unsigned DEFAULT NULL,
+        application_uuid varchar(36) NOT NULL,
```

Comment:
Thought was to add this in for future Brizo users but you are correct, newly created migrations would be correct.